client/question/question.js

```diff
@@ -550,101 +550,195 @@ tie.constant(
   'PREREQ_CHECK_TYPE_INVALID_STUDENTCODE_CALL', 'invalidStudentCode');
 
 /**
-  Dictionary of the customized runtime error feedback messages
-  - `checker` function checks to see if the error matches with the type of
-    feedback message
-  - `generateMessage` function returns the appropriate feedback message for the
-    given error
+ * Dictionary of the novice-friendly syntax error feedback messages
+ *
+ * `friendlyErrorCheck` function checks to see if the error matches with the
+ * type of feedback message
+ *
+ * `getFriendlyErrorText` function returns the appropriate feedback message for
+ * the given error
+ *
+ * @type {Object}
+ */
+tie.constant('FRIENDLY_SYNTAX_ERROR_TRANSLATIONS', {
+  python: [
+    {
+      friendlyErrorCheck: function(errorString) {
+        var re = new RegExp('^NameError: global name .* is not defined$');
+        return re.test(errorString);
+      },
+      getFriendlyErrorText: function(errorString) {
+        var re = new RegExp('NameError: global name (.*) is not defined');
+        var varName = errorString.match(re)[1];
+        return [
+          '<code>', varName, '</code> appears to be a name that was not ',
```

Comment:
There are other parts of the text that uses <code> as well, e.g., ' = []' and '= \'\''. I feel like it's more readable as is rather than creating multiple variables.

Reply:
I disagree, but only slightly. I'm happy for you to make the call.